src/app/modules/form/form.errors.ts

```diff
@@ -17,13 +17,19 @@ export class FormDeletedError extends ApplicationError {
  * that is to be returned for public consumption when its status is PRIVATE.
  */
 export class PrivateFormError extends ApplicationError {
+  /** Extra meta for form title. */
+  formTitle: string
+
   /**
    * @param message Message used should be the form's inactive message.
+   * @param formTitle Extra meta for form title
    */
   constructor(
     message = 'If you think this is a mistake, please contact the agency that gave you the form link.',
+    formTitle: string,
   ) {
     super(message)
+    this.formTitle = formTitle
```

Comment:
actually why are we shoving so much info into a custom error object? shouldn't we just let errors be dumb? seems a bit like incurring tech debt for the sake of logging convenience

Reply:
This is more than logging convenience, the frontend actually uses it to display context